Modules/ModuleFactory.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using System.Linq;
+using Ovation.FasterQC.Net.Utils;
+
+namespace Ovation.FasterQC.Net.Modules
+{
+    public static class ModuleFactory
+    {
+        private static readonly Dictionary<string, IQcModule> moduleMap = new()
+        {
+            ["BasicStatistics"] = new BasicStatistics(),
```

Comment:
oooo, that's an interesting syntax. i hadn't seen that one yet

Reply:
yeah, that's the "new" key value pair literal syntax.